packages/dd-trace/src/appsec/index.js

```diff
@@ -137,6 +139,17 @@ function onRequestBodyParsed ({ req, res, abortController }) {
   handleResults(results, req, res, rootSpan, abortController)
 }
 
+function onNextRequestBodyParsed ({ req, body }) {
```

Comment:
Remember what we decided a while ago: it should be the responsability of the publisher to format the data in a standardized way, so we can have the same listeners for multiple instrumentations. i.e. there should be only one `onBodyParsed()` and it should support being called without a valid `res`/`abortController` to not block when they're not available.

Reply:
oh yeah i didn't even realize they were sourcing from req, that's definitely a mistake